src/pages/index.js

```diff
@@ -65,34 +166,65 @@ consts.profileEditButton.addEventListener("click", () => {
 consts.profileAddButton.addEventListener("click", () => {
   addCardModal.open();
 });
+consts.avatarContainer.addEventListener("click", () => {
+  avatarEditModal.open();
+});
 
-// validation
-
-const addCardValidator = new FormValidator(
-  consts.validatorConfig,
-  consts.addCardModalDiv,
-);
-
-addCardValidator.enableValidation();
-
-const profileEditValidator = new FormValidator(
-  consts.validatorConfig,
-  consts.profileEditDiv,
-);
+// initial loading handlers
 
-profileEditValidator.enableValidation();
+function handleProfileLoad({ name, about, avatar }) {
+  profileEl.setUserInfo(name, about);
+  profileEl.setAvatar(avatar);
+}
+function loadAll() {
+  api
+    .initialLoad()
+    .then((data) => {
+      handleProfileLoad(data[0]);
+      return data;
+    })
+    .then((data) => {
+      loadDefaultCards(data[1]);
+    });
+}
+function loadDefaultCards(cardsArr) {
+  if (cardsArr.length != 0) {
+    gallery["items"] = cardsArr;
+    gallery.renderItems();
+  } else {
+    console.log("Gallery empty. Loading default cards...");
+    gallery["items"] = addDefaultCards();
+    gallery.renderItems();
+  }
+}
+function addDefaultCards() {
+  const defaultCards = [];
+  consts.initialCards.forEach((card) => {
+    api.addCard(card);
+    defaultCards.push(card);
+  });
+  return defaultCards;
+}
 
+// validation
+const formValidation = [
+  new FormValidator(consts.validatorConfig, consts.addCardModalDiv),
+  new FormValidator(consts.validatorConfig, consts.profileEditDiv),
+  new FormValidator(consts.validatorConfig, consts.avatarEditDiv),
```

Comment:
Please, don't add validators in an **array**. It's difficult to manage them if there are 10 validators


You need to use either object or separate variables